src/public/map.js

```diff
@@ -193,9 +135,74 @@ function fail(jqXHR, status, errorThrown) {
   console.log(errorThrown);
 }
 
+function displayFeatures(features) {
+  console.log(features);
```

Comment:
just for short-term debugging? if so, please remove; otherwise, have the log provide some context